ibis/impala/tests/test_window.py

```diff
@@ -107,6 +108,12 @@ def test_add_default_order_by(alltypes):
             ibis.trailing_window(10),
             'rows between 10 preceding and current row',
         ),
+        (
+            ibis.trailing_window(
+                rows_with_max_lookback(3, ibis.interval(days=3))
+            ),
+            'rows between 3 preceding and current row',
```

Comment:
I'll create a follow up issue for this as well, but I don't think we should implement this for any backends unless we can give it well-defined behavior.